e2e/cypress/tests/integration/enterprise/self-hosted/self_hosted_purchase_spec.ts

```diff
@@ -0,0 +1,382 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// e.g. not_cloud cloud because we always want to exclude running automatically
+// until we create the special self-hosted run setup
+// Stage: @prod
```

Comment:
Should this be `@prod` if we can only run this locally at the time?

Reply:
Good catch! It should be `@dev` for now.